lib/chef/provisioning/vsphere_driver/driver.rb

```diff
@@ -186,7 +186,11 @@ def find_or_create_vm(bootstrap_options, machine_spec, action_handler)
     def full_description(machine_spec, bootstrap_options)
       description = [ "creating machine #{machine_spec.name} on #{driver_url}" ]
       bootstrap_options.to_hash.each_pair do |key,value|
-        description << "  #{key}: #{value.inspect}"
+	if value.is_a?(Hash) then
+          temp_value = value.clone
+          temp_value[:password] = "*********" if value.has_key?(:password)
+        end
```

Comment:
@devilldog one thing occurs to me, that `temp_value` will ONLY be initialized if `value` is a `Hash`.  Thus `description` will end up missing all other non-Hash values.

I think this would solve that problem:
```
if value.is_a?(Hash) then
   temp_value = value.clone
   temp_value[:password] = "*********" if value.has_key?(:password)
else
   temp_value = value
end
description << "  #{key}: #{temp_value.inspect}"
```